turnt.py

```diff
@@ -300,7 +301,11 @@ def run_test(path, config_name, idx, save, diff, verbose, dump, args=None):
             os.unlink(stderr.name)
 
 
-@click.command()
+def cli():
+      pass
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
```

Comment:
I would love to put this inline because it's only relevant to the Click setup:
```suggestion
@click.command(context_settings={'help_option_names': ['-h', '--help']})
```